plugins/inputs/ping_native/README.md

```diff
@@ -0,0 +1,68 @@
+# Ping Native Input Plugin
+
+Sends a ping message and reports the results. This is done in pure go, eliminating the need to execute the system `ping` command.
+
+There is currently no support for TTL on windows, track progress at https://github.com/golang/go/issues/7175 and https://github.com/golang/go/issues/7174
+
+
+### Configuration:
+
+```toml
+[[inputs.ping_native]]
```

Comment:
Merge this with the ping plugin, since we keep the same API: config and metric format.  Easiest way I think will be to pull the whole old implementation out and either call it or this one from a top plugin, ala:
```
func (p *Ping) Gather(acc telegraf.Accumulator) error {
	if p.Method == "native" {
		return p.native.Gather(acc)
	} else {
		return p.exec.Gather(acc)
	}
}